src/fontra/client/core/glyph-svg.js

```diff
@@ -51,3 +51,20 @@ function formatCoordinate(x, y, numerator, denominator, dx, dy) {
   y = Math.round(y * numerator + dy) / denominator;
   return `${x},${y}`;
 }
+
+export function pathToSVG(path, bounds) {
+  const svgPath = new SVGPath2D();
+  path.drawToPath2d(svgPath);
+  const pathString = svgPath.getPath();
+
+  const width = bounds["xMax"] - bounds["xMin"];
```

Comment:
We need to do a slightly bigger change here. Something along these lines:
```js
  const xMin = Math.floor(bounds.xMin);
  const yMin = Math.floor(bounds.yMin);
  const xMax = Math.floor(bounds.xMax);
  const yMax = Math.floor(bounds.yMax);
  const width = xMax - xMin;
  const height = yMax - yMin;
```
Then, the viewbox will be (bear with me): xMin, 0, xMax, height
And the sixth matrix argument will be yMax.

Reply:
I see your recent change, but please also do:
> Then, the viewbox will be (bear with me): xMin, 0, xMax, height
> And the sixth matrix argument will be yMax.

(Please check for unused variables: at least `width` will remain unused I think, but perhaps others.)

Reply:
Width is used for width attribute of the svg, so all good.